resource/lvm/lowlevel/exec_test.go

```diff
@@ -0,0 +1,42 @@
+package lowlevel_test
+
+import (
+	"github.com/asteris-llc/converge/resource/lvm/lowlevel"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestExecRun(t *testing.T) {
```

Comment:
It might be easier to follow if you have one Test[FuncName], and then use t.Run to test the different cases of success and failure resulting in error. Using t.Run is what we are trying to do now, so updating throughout is probably a good idea.